packages/docs/examples/AddNetwork.tsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-console */
+import { cssObj } from '@fuel-ui/css';
+import { Box, Button, Input, Text } from '@fuel-ui/react';
+import { useState } from 'react';
+
+// eslint-disable-next-line import/no-relative-packages
+import type { Network } from '../../types/src';
+import { ExampleBox } from '../src/components/ExampleBox';
+import { useFuel } from '../src/hooks/useFuel';
+import { useIsConnected } from '../src/hooks/useIsConnected';
+import { useLoading } from '../src/hooks/useLoading';
+
+export function AddNetwork() {
+  const [fuel, notDetected] = useFuel();
+  const [isConnected] = useIsConnected();
+  const [network, setNetwork] = useState<Network>({
+    name: '',
+    url: '',
+  });
```

Comment:
can we use Localhost as default network ?

```suggestion
  const [network, setNetwork] = useState<Network>({
    name: 'Localhost',
    url: 'http://localhost:4000/graphql',
  });
```